### PR TITLE
feat: added snackbar and csv file delete option

### LINKED ISSUE #1217: show the path of file and delete option after csv recording
**Actual Behaviour**

Currently, a toast message is generated saying data recording stopped after CSV file is saved, 

**Expected Behaviour**

there should be path shown to the user where the file has been saved and also if user recorded the data by mistake then there should be option to delete the file after it has been saved.

**Steps to reproduce it**

LuxMeterActivity -> Create a snackbar showing the path and delete option to delete current file

**Would you like to work on the issue?**

Yes

### PR DESCRIPTION
**Pull request by Avjeet**
Fixes #1217 

**Changes**: 
- Added snackbar to show the path where the csv file is stored in the external storage
- Added delete option in snack bar to delete the file if not required

**Screenshot/s for the changes**: 
image

image

**Checklist**: [Please tick following check boxes with `[x]` if the respective task is completed]
- [x] I have used resources from `strings.xml`, `dimens.xml` and `colors.xml` without hard-coding them
- [x] No modifications done at the end of resource files `strings.xml`, `dimens.xml` or `colors.xml`
- [x] I have reformatted code in every file included in this PR [<kbd>CTRL</kbd>+<kbd>ALT</kbd>+<kbd>L</kbd>]
- [x] My code does not contain any extra lines or extra spaces
- [x] I have requested reviews from other members

**APK for testing**: 
[snackbar.zip](https://github.com/fossasia/pslab-android/files/2188477/snackbar.zip)

### CODE REVIEW

File: app/src/main/java/org/fossasia/pslab/fragment/LuxMeterFragmentData.java [hunk 1]
```diff
@@ -286,6 +290,26 @@ private void visualizeData() {
                             lux_logger.writeCSVFile(data);
                         }
                         gpsLogger.removeUpdate();
+                        CustomSnackBar.showSnackBar((CoordinatorLayout) parent.findViewById(R.id.cl),
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/fragment/LuxMeterFragmentData.java]: This file needs some reformatting!

File: app/src/main/java/org/fossasia/pslab/others/CSVLogger.java [hunk 2]
```diff
@@ -89,4 +90,15 @@ public void writeCSVFile(String data) {
             }
         }
     }
+
+    public String getCurrentFilePath() {
+        String s = Environment.getExternalStorageDirectory().getAbsolutePath() +
+                File.separator + CSV_DIRECTORY + File.separator + category;
+        Log.d("pathDir", s);
```
Comment (CloudyPadmal) [inline app/src/main/java/org/fossasia/pslab/others/CSVLogger.java]: Log is not needed once you test it

**Comment by Avjeet** [comment]
@CloudyPadmal changes done

File: app/src/main/res/layout/activity_lux_main.xml [hunk 3]
```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
     android:layout_width="match_parent"
-    android:layout_height="match_parent">
+    android:layout_height="match_parent"
+    android:id="@+id/cl">
```
Comment (abhinavraj23) [inline app/src/main/res/layout/activity_lux_main.xml]: @Avjeet This file also needs reformat as id should come before height

File: app/src/main/res/values/styles.xml [hunk 4]
```diff
@@ -11,14 +11,19 @@
...
+
     <style name="AppThemeSplash" parent="AppTheme">
         <item name="colorPrimaryDark">@color/darkWhite</item>
     </style>
-    <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.AppCompat.Dark.ActionBar"/>
-    <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light"/>
+
+    <style name="AppTheme.AppBarOverlay" parent="ThemeOverlay.AppCompat.Dark.ActionBar" />
+
+    <style name="AppTheme.PopupOverlay" parent="ThemeOverlay.AppCompat.Light" />
```
Comment (abhinavraj23) [inline app/src/main/res/values/styles.xml]: @Avjeet why are we having extra spaces here? Please remove them.

**Comment by Avjeet** [comment]
@abhinavraj23 changes done, plz review

**Pull request merged**